### ISSUE TITLE
Edit Tags – Page Title for each tab are not unique 

### ISSUE THREAD

**Issue by hydegeoff** [mmiron-HC/WST-RSAMS#151]
URL: https://rsams.dev.cloud.openplus.ca/en/user/
User Agent: Chrome
Language: EN and FR

Issue Description: 
When switching tab 4 of 7 on the Edit Tags page have “Content | RSAMS” as its page title, the page title should be more specific.  Something like Content – [tab label] | RSAMS would be more accurate.  The H1 should be updated respectively as well.  This helps users with screen readers, but it also helps visual users as you can see in the screenshot, a visual user has no idea that each browser tab are on different pages.
![Edit tags - Page Title is same for four tabs](https://user-images.githubusercontent.com/88147013/161620023-db7fc6f8-a09e-4b9b-9a84-6a9cb5b5e6a9.JPG)

Steps to reproduce:
1.	Login to RSAMs
2.	Select the [User name]
3.	Select Edit tags and navigate to the following tabs:
•	Alerts/Recalls
•	Locked
•	A/R migration dept
•	A/R migration
Problem: They all have the same page title 

Expected results:
Page titles should be unique

**Comment by davidgotlieb** [comment]
I think the risk here is pretty low since this page is deep in the top level admin. Only a handful of people will have access to this section. Not saying it should be on the list, but we definitely should prioritize fixing more impactful issues.

**Comment by hydegeoff** [comment]
This issue impacts all users, but it has greater impact on user with screen readers.

**Comment by davidgotlieb** [comment]
> This issue impacts all users, but it has greater impact on user with screen readers.

I understand @hydegeoff , My point is that not all users have access to that section of the interface. That's an advanced section that only top level admins can access (Me, Eric, and maybe 4 other people), so risk is pretty low that it will affect anyone significantly.

**Comment by hydegeoff** [comment]
When we say "all-users", it is always in the context of all the different user types who have access to the specific page.

**Comment by echouin** [comment]
this is not relevant.. Only the most advanced vendor has access.
